planetarium/NineChronicles
# Fix Item Recipe and sub-recipe for craft V2

### Description

1. Renew all sub recipes
2. Renew Stage 1~200 item drops

### How to test

1. Check all the recipe's materials
2. Check all the stage's drop tables

### Related Links

### Required Reviewers

@Namyujeong @boscohyun 

### Screenshot
